src/core/panel/Panel.ts

```diff
@@ -321,13 +321,13 @@ class Panel {
     if (cached) {
       this._size = cached.size;
       this._margin = { ...cached.margin };
-      this._height = horizontal ? getElementSize({
+      this._height = cached.height ? cached.height : getElementSize({
```

Comment:
This can result in a recalculation of height when `cached.height` equals 0.
Please use `??` instead

Reply:
I think `??` may have issues with Flicking's browser compatibility.
There is "IE 9+(With polyfill), 11+ for Angular & Svelte" in supported browsers, [but it looks like we won't be able to use ?? in these versions.](https://caniuse.com/?search=%3F%3F)